debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -241,6 +263,24 @@ public void dispatchSchemaChangeEvent(T dataCollectionId, SchemaChangeEventEmitt
         schemaChangeEventEmitter.emitSchemaChangeEvent(new SchemaChangeEventReceiver());
     }
 
+    public void dispatchSchemaChangeEvent(List<T> dataCollectionIds, SchemaChangeEventEmitter schemaChangeEventEmitter) throws InterruptedException {
```

Comment:
I don't find any callers to this method.

Reply:
@gunnarmorling This one is oging to be used when MySQL is migrated to the new framework.